Port the `ImageClassifier` guide to `keras_core`

tirthasheshpatel opened the pull request:
This PR ports the [ImageClassifier guide on keras.io](https://keras.io/guides/keras_cv/classification_with_keras_cv/) to `keras_core`. It uses the `keras_cv.backend` module to import Keras which provides support for both `tf.keras` and `keras_core` in a backend-agnostic manner.

To run the guide on Colab, use the [`core` branch from Ian's fork](https://github.com/ianstenbit/keras-cv/tree/core) of Keras-CV where we are porting backbones to `keras_core`.

cc @ianstenbit

fchollet commented on examples/keras_io/vision/image_classifier.py:
```diff
@@ -0,0 +1,679 @@
+    https://colab.research.google.com/drive/1qvQbaRuuMKJzp7cYgEQ7lY6kvBzglIqj
+
+# Classification with KerasCV
+
```
Remove the first few lines above

tirthasheshpatel replied:
Done.

fchollet commented on examples/keras_io/vision/image_classifier.py:
```diff
@@ -0,0 +1,679 @@
+
+# Classification with KerasCV
+
+**Author:** [lukewood](https://lukewood.xyz)<br>
```
This is using markdown formatting -- don't. See other files for examples

tirthasheshpatel replied:
Done in #608.

fchollet commented on examples/keras_io/vision/image_classifier.py:
```diff
@@ -0,0 +1,679 @@
+
+## Multi-Backend Support
+
+Keras-CV's `ImageClassifier` model supports several backends like Jax, PyTorch, and TensorFlow with the help of `keras_core`. To enable multi-backend support in Keras-CV, set the `KERAS_CV_MULTI_BACKEND` environment variable. We can then switch between different backends by setting the `KERAS_BACKEND` environment variable. Currently, `tensorflow`, `jax`, and `torch` are supported.
```
- Keep lines under 80 chars
- Refer to KerasCV as "KerasCV"

tirthasheshpatel replied:
Done in #608 with the exception of a few links that can't be wrapped to fit in 80 chars character limit.

fchollet reviewed (commented):
Thanks for the PR!

tirthasheshpatel commented:
@fchollet I don't have the permission to push to the upstream branch directly anymore, so, I have created a #608 on this branch addressing your comments here.

fchollet commented:
Let's carry on in the other PR then!
